lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp

```diff
@@ -3,7 +3,7 @@
 #include <ATen/core/Reduction.h>
 #include <torch/csrc/jit/frontend/sugared_value.h>
 #include <torch/jit.h>
-#include <torch/torch.h>
+// #include <torch/torch.h>
```

Comment:
Yesterday I saw a weird bug in the CI that <torch/torch.h> and <torch/csrc/jit/frontend/sugared_value.h> point to the same file from different install locations, and therefore introduce a redefinition bug. That's why I comment out this header to see if it fixes the issue. Here is a chopped version:
```
Sep 15 01:10:26 In file included from /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/nn/module.h:6:0,
Sep 15 01:10:26                  from /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/nn/cloneable.h:3,
Sep 15 01:10:26                  from /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/nn.h:3,
Sep 15 01:10:26                  from /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/all.h:13,
Sep 15 01:10:26                  from /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/torch.h:3,
Sep 15 01:10:26                  from lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp:6:
Sep 15 01:10:26 /opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/ordered_dict.h:13:7: error: redefinition of 'class torch::OrderedDict<Key, Value>'
Sep 15 01:10:26  class OrderedDict {
Sep 15 01:10:26        ^~~~~~~~~~~
Sep 15 01:10:26 In file included from /var/lib/jenkins/workspace/torch/csrc/jit/api/module.h:13:0,
Sep 15 01:10:26                  from /var/lib/jenkins/workspace/torch/csrc/jit/frontend/sugared_value.h:8,
Sep 15 01:10:26                  from lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp:4:
Sep 15 01:10:26 /var/lib/jenkins/workspace/torch/csrc/api/include/torch/ordered_dict.h:13:7: note: previous definition of 'class torch::OrderedDict<Key, Value>'
Sep 15 01:10:26  class OrderedDict {
``` 

It's weird that <torch/torch.h> is installed at `/opt/conda/lib/python3.6/site-packages/torch/include/torch/csrc/api/include/torch/torch.h` while <torch/csrc/jit/frontend/sugared_value.h> is installed at `/var/lib/jenkins/workspace/torch/csrc/jit/frontend/sugared_value.h`. Wondering if someone can explain this situation to me and decide if the deleting <torch/torch.h> is a good solution.

P.S. A detailed report: https://app.circleci.com/pipelines/github/pytorch/pytorch/380340/workflows/84296af5-fee4-4812-95d3-419e097047a6/jobs/16023443.